src/Package/Impl/Repl/Commands/ReplShortcutSetting.cs

```diff
@@ -2,91 +2,39 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
-using System.Diagnostics;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
 using EnvDTE;
-using Microsoft.R.Editor.Settings;
 using Microsoft.VisualStudio.R.Package.Shell;
 
 namespace Microsoft.VisualStudio.R.Package.Repl.Commands {
     internal static class ReplShortcutSetting {
-        private const string CommandName = "EditorContextMenus.CodeWindow.ExecuteLineInInteractive";
-
-        private static bool _currentSetting;
+        private static Dictionary<string, string> _editorShortcutRemaps = new Dictionary<string, string>() {
+            { "Edit.LineOpenAbove", "Text Editor::ctrl+k,ctrl+o" }, // Make Ctrl+Enter work in R Editor
```

Comment:
It feels like we're removing choice here, especially for those who want to use VS as a true multi-language IDE, and not just for R. It will also make it more complicated to unify with PTVS in one box later, because shortcuts are all over the place (whereas our default shortcuts before this change were more in line with VS and PTVS practice). 

Also, I feel like we might be hearing negative feedback from those who didn't find the shortcuts, but we are not hearing positive feedback from those who did (and who might like/prefer them), simply because they don't see any point in spelling it out. 

Either way, whatever is the default, I think we do need a profile for multi-language VS use that is consistent with VS (generic profile) rather than RS.